geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -460,108 +478,110 @@ static IProcess projectedFacadeAreaDistribution() {
  * between buildings and RSU
  * @param rsuTable the name of the input ITable where are stored the RSU
  * @param prefixName String use as prefix to name the output table
- * @param listLayersBottom the list of height corresponding to the bottom of each vertical layers (default [0, 10, 20, 30, 40, 50])
+ * @param listLayersBottom the list of height corresponding to the bottom of each vertical layers (default [0, 10, 20,
+ * 30, 40, 50])
  *
  * @return outputTableName Table name in which the rsu id and their corresponding indicator value are stored
  *
  * @author Jérémy Bernard
  */
 static IProcess roofAreaDistribution() {
+    def final GEOMETRIC_COLUMN_RSU = "the_geom"
+    def final GEOMETRIC_COLUMN_BU = "the_geom"
+    def final ID_COLUMN_RSU = "id_rsu"
+    def final ID_COLUMN_BU = "id_build"
+    def final HEIGHT_WALL = "height_wall"
+    def final HEIGHT_ROOF = "height_roof"
+    def final BASE_NAME = "rsu_roof_area_distribution"
+
     return processFactory.create(
             "RSU roof area distribution",
-            [rsuTable: String, buildingTable: String, listLayersBottom: double[],
-             prefixName: String, datasource: JdbcDataSource],
+            [rsuTable: String, buildingTable: String, listLayersBottom: [0, 10, 20, 30, 40, 50], prefixName: String,
+                    datasource: JdbcDataSource],
             [outputTableName : String],
-            { rsuTable, buildingTable, listLayersBottom = [0, 10, 20, 30, 40, 50],
-              prefixName, datasource ->
+            { rsuTable, buildingTable, listLayersBottom, prefixName, datasource ->
 
                 logger.info("Executing RSU roof area distribution")
 
-                def geometricColumnRsu = "the_geom"
-                def geometricColumnBu = "the_geom"
-                def idColumnRsu = "id_rsu"
-                def idColumnBu = "id_build"
-                def height_wall = "height_wall"
-                def height_roof = "height_roof"
-
                 // To avoid overwriting the output files of this step, a unique identifier is created
-                def uid_out = UUID.randomUUID().toString().replaceAll("-", "_")
-
                 // Temporary table names
-                def buildRoofSurfIni = "build_roof_surf_ini" + uid_out
-                def buildVertRoofInter = "build_vert_roof_inter" + uid_out
-                def buildVertRoofAll = "buildVertRoofAll" + uid_out
-                def buildRoofSurfTot = "build_roof_surf_tot" + uid_out
+                def buildRoofSurfIni = "build_roof_surf_ini" + uuid()
```

Comment:
Is uuid() a function that you have coded for the need of geoclimate ? Or an existing one ?